framework/src/wifi_manager/wifi_profile.c

```diff
@@ -0,0 +1,238 @@
+#include <tinyara/config.h>
```

Comment:
You should add Samsung copyright and license at top of file.

Reply:
Ok I will add